src/styles/main.scss

```diff
@@ -1,3 +1,13 @@
+@import "./blocks/calendar.scss";
+@import "./utils/variables.scss";
+
 body {
   margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+@for $index from 1 through 31 {
```

Comment:
It's better to generate the numbers for the days inside the calendar block or element instead of using a global div selector. This way, the code is more maintainable and follows BEM methodology better.